test/and.test.js

```diff
@@ -1,4 +1,4 @@
-const { and } = require("../lib");
```

Comment:
We want all functions to be exported from lib/index.js, the and method is missing there, which is why this could not work. Please add the export in the index file and switch back to the old import :)